### PR TITLE
Remove time_now() - time_now_d() is the replacement. 

### PR DESCRIPTION
**Pull request by hrydgard**
Absolute time values can't be stored reliably in floats, just not enough precision. Let's not pretend they can - mostly to prevent future mistakes as none of these uses seem particularly dangerous.

### CODE REVIEW

File: Common/Vulkan/VulkanMemory.cpp [hunk 1]
```diff
@@ -278,7 +278,7 @@ bool VulkanDeviceAllocator::AllocateFromSlab(Slab &slab, size_t &start, size_t b
 
 	// Remember the size so we can free.
 	slab.allocSizes[start] = blocks;
-	slab.tags[start] = { tag, time_now(), 0.0f };
+	slab.tags[start] = { tag, (float)time_now_d(), 0.0f };
```
Comment (unknownbrackets) [inline Common/Vulkan/VulkanMemory.cpp:281]: Hm, should really make both of these doubles.  There aren't that many of these so it should be fine.

-[Unknown]

**Review by unknownbrackets — APPROVED** [review]

File: Common/Vulkan/VulkanMemory.cpp (hunk 1 above)
Comment (hrydgard) [reply Common/Vulkan/VulkanMemory.cpp:281]: I was worried about savestates and thought it couldn't matter much here (but got too lazy to actually check whether these are saved heh).

Comment (hrydgard) [reply Common/Vulkan/VulkanMemory.cpp:281]: Come to think of it, this being the Vulkan slab allocator, these definitely aren't saved. I'll change t.

Comment (unknownbrackets) [reply Common/Vulkan/VulkanMemory.cpp:281]: I've definitely erred on the side of savings things unnecessarily, but even I wouldn't save the slabs, heh.

Although I have been thinking of saving the framebufs for reverse debugging...

-[Unknown]

**Pull request merged**